gnovm/pkg/gnolang/machine.go

```diff
@@ -1900,7 +1900,11 @@ func (m *Machine) PushFrameCall(cx *CallExpr, fv *FuncValue, recv TypedValue, is
 				// Neither cross nor didswitch.
 				recvPkgOID := ObjectIDFromPkgID(recvOID.PkgID)
 				objpv := m.Store.GetObject(recvPkgOID).(*PackageValue)
-				rlm = objpv.GetRealm()
+				if objpv.IsRealm() && objpv.Realm == nil {
+					rlm = m.Store.GetPackageRealm(objpv.PkgPath)
+				} else {
+					rlm = objpv.Realm
+				}
```

Comment:
// get package associated realm if nil.

Reply:
Seems good to me, but I'm not sure about the expected behavior. It's better for someone more knowledgeable than me on the crossing topic to approve.

Reply:
maybe there should be a convenient way to get the owner real for an object? i guess it's enough for the fix now.